packages/launcher-client/src/client/impl/default.launcher.client.ts

```diff
@@ -156,8 +157,12 @@ export default class DefaultLauncherClient implements LauncherClient {
     if (config.gitProvider) {
       headers['X-Git-Provider'] = config.gitProvider;
     }
-    if (authorizationToken) {
+    if (typeof authorizationToken === 'string') {
       headers['Authorization'] = `Bearer ${authorizationToken}`;
+    } else if (authorizationToken) {
+      const authToken = authorizationToken as AuthorizationToken[];
+      authToken.map(t => headers[t.header] = `Bearer ${t.token}`);
+      headers['Authorization'] = 'Bearer eyJhbGciOiJIUzI1NiJ9.e30.ZRrHA1JJJW8opsbCGfG_HACGpVUMN_a9IV7pAx_Zmeo';
```

Comment:
@edewit this is until @gastaldi changes the backend right?

Reply:
Right, the backend always needs an `Authorization` header even if we are sending an `X-OpenShift-Authorization`

Reply:
@gastaldi when are you going to change that?